src/dagre/position/bk.ts

```diff
@@ -46,7 +46,7 @@ function findType1Conflicts(g, layering) {
       k0 = 0,
       // Tracks the last node in this layer scanned for crossings with a type-1
       // segment.
-      scanPos = 0 as number | string,
+      scanPos: number | string = 0,
```

Comment:
I meant this more as a general tip. It’s good you chaned it here, but it occurs in more places. Typically type casts should be rare.